### PR TITLE
Fix create an app

### PR DESCRIPTION
**Pull request by floreks**
There was an issue with creating an app when user had no permission to check if object exists but was able to create new objects.

### CODE REVIEW

File: src/app/frontend/deploy/deployfromsettings/deployfromsettings.html [hunk 1]
```diff
@@ -178,7 +178,7 @@
       </div>
 
       <kd-user-help>
-        [[The specified labels will be applied to the created Replication Controller, Service (if any) and Pods. Common labels include release, environment, tier, partition and track.|User help for the labels section on the deploy from settings page.]]
+        [[The specified labels will be applied to the created Deployment, Service (if any) and Pods. Common labels include release, environment, tier, partition and track.|User help for the labels section on the deploy from settings page.]]
```
Comment (maciaszczykm) [inline src/app/frontend/deploy/deployfromsettings/deployfromsettings.html]: Generate translations.

File: src/app/backend/errors/handler.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ import (
 	"log"
 	"net/http"
 
+	"github.com/emicklei/go-restful"
```
Comment (maciaszczykm) [inline src/app/backend/errors/handler.go]: Using `restful "github.com/emicklei/go-restful"` will let us avoid problems with `gulp format`.

Comment (floreks) [reply src/app/backend/errors/handler.go]: For me it didn't really make any difference. Sometimes import gets still deleted. Maybe IntelliJ plugin removes aliases and then gulp format removes import.

Comment (maciaszczykm) [reply src/app/backend/errors/handler.go]: My IntelliJ leaves them prefixed, let's use this format.

**Pull request merged**